### PR TITLE
fix: open in colab

### PR DESCRIPTION
**Pull request by ydcjeff**
### Description

fix colab not opening
ref #103 

### Additional context

---

### What is the purpose of this pull request?

- [x] Bug fix
- [ ] New feature
- [ ] Other

### CODE REVIEW

**Comment by trsvchn** [comment]
When I press "Colab" button I have a redirection to a new tab with the following url:
`https://deploy-preview-162--code-generator.netlify.app/undefined`

File: src/components/NavColab.vue [hunk 1]
```diff
@@ -109,119 +80,140 @@ export default {
...
+          //   cells: [],
+          //   nbformat: 4,
+          //   nbformat_minor: 2
+          // }
 
-          var firstCell // README
-          var reqCell // !pip install ...
-          var configCell // %%writefile config.yaml
-          var lastCell // main()
+          // var firstCell // README
```
Comment (trsvchn) [inline src/components/NavColab.vue:111]: Can we remove these commented lines? I think we do not need them.

**Review by trsvchn — APPROVED** [review]
Looks amazing! Many thanks @ydcjeff !

File: src/components/NavColab.vue (hunk 1 above)
Comment (trsvchn) [reply src/components/NavColab.vue:111]: I mean all of them

**Comment by vfdev-5** [comment]
Great job, Jeff ! Maybe we could add the following to the metadata to use GPU by default
```
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "colab": {
      "name": "pytorch-ignite-notebook.ipynb",
      "provenance": []
    },
    "accelerator": "GPU"
  },
```
and the first cell with 
```
# <template name> by PyTorch-Ignite Code-Generator

Please, run the cell below to execute your code.
```

**Comment by trsvchn** [comment]
> Great job, Jeff ! Maybe we could add the following to the metadata to use GPU by default
> 
> ```
>   "metadata": {
>     "kernelspec": {
>       "display_name": "Python 3",
>       "name": "python3"
>     },
>     "colab": {
>       "name": "pytorch-ignite-notebook.ipynb",
>       "provenance": []
>     },
>     "accelerator": "GPU"
>   },
> ```
> 
> and the first cell with
> 
> ```
> # <template name> by PyTorch-Ignite Code-Generator
> 
> Please, run the cell below to execute your code.
> ```

And colab selects the GPU runtime for us?

**Comment by vfdev-5** [comment]
> > Great job, Jeff ! Maybe we could add the following to the metadata to use GPU by default
> > ```
> >   "metadata": {
> >     "kernelspec": {
> >       "display_name": "Python 3",
> >       "name": "python3"
> >     },
> >     "colab": {
> >       "name": "pytorch-ignite-notebook.ipynb",
> >       "provenance": []
> >     },
> >     "accelerator": "GPU"
> >   },
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and the first cell with
> > ```
> > # <template name> by PyTorch-Ignite Code-Generator
> > 
> > Please, run the cell below to execute your code.
> > ```
> 
> And colab will start the GPU runtime for us?

yes

**Comment by trsvchn** [comment]
Thanks! Didn't know that!

**Pull request merged**